pkg/build/api/types.go

```diff
@@ -199,6 +199,9 @@ type STIBuildStrategy struct {
 	// Image is the image used to execute the build.
 	Image string `json:"image,omitempty"`
 
+	// From is reference to an image repository from where the docker image should be pulled
+	From kapi.ObjectReference `json:"from,omitempty"`
```

Comment:
Actually we've settled on `From` taking precedence over `Image` as this is the preferred for us.